apps/checkout-app/types/common.ts

```diff
@@ -13,6 +17,7 @@ export type PaymentMethodID = "creditCard" | "applePay" | "paypal";
 export interface PaymentMethod {
   id: PaymentMethodID;
   name: string;
+  logo?: OverridableComponent<SvgIconTypeMap<{}, "svg">>;
```

Comment:
I think since we use this in multiple places it'd make sense to keep this type `OverridableComponent<SvgIconTypeMap<{}, "svg">>` as `IconComponent`